apps/pigment-css-next-app/src/app/material-ui/react-backdrop/page.tsx

```diff
@@ -0,0 +1,16 @@
+'use client';
+import * as React from 'react';
+import SimpleBackdrop from '../../../../../../docs/data/material/components/backdrop/SimpleBackdrop';
+
+export default function Backdrop() {
+  return (
+    <React.Fragment>
+      <section>
+        <h2> Simple Backdrop</h2>
+        <div className="demo-container">
+          <SimpleBackdrop />
+        </div>
+      </section>
+    </React.Fragment>
```

Comment:
```suggestion
    <section>
      <h2> Simple Backdrop</h2>
      <div className="demo-container">
        <SimpleBackdrop />
      </div>
    </section>
```

This diff should fix the lint error. We could also update the script, but I don't think it's worth the effort :)

Reply:
Resolved 🙌🏼